tooling/ci/demos-deploy.sh

```diff
@@ -19,8 +19,13 @@ if [[ " ${releaseBranches[@]} " =~ " ${CIRCLE_BRANCH} " ]]; then
   # Clone the demo repo
   cd $HOME
   git clone git@github.com:GetTerminus/ui-demos-$CIRCLE_BRANCH.git
+
+  # Enter the new repo
   cd ui-demos-$CIRCLE_BRANCH
 
+  # Remove old demo files
+  find -E . -regex '.*\.(html|css|js)' -type f -delete
```

Comment:
how do you differentiate between old and new files?

Reply:
It's all old. The only reason I don't just empty the entire directory is so that our favicon and 3rdpartylicenses file stays intact since I _think_ those aren't in the dist/ automatically

Reply:
Fair enough. Updated the comment to say `Enter the demos repo` for clarity.